helm-chart/ray-cluster/templates/raycluster-cluster.yaml

```
@@ -92,6 +92,10 @@ spec:
         affinity: {{- toYaml .Values.head.affinity | nindent 10 }}
         tolerations: {{- toYaml .Values.head.tolerations | nindent 10 }}
         nodeSelector: {{- toYaml .Values.head.nodeSelector | nindent 10 }}
+        {{- with .Values.head.podSecurityContext }}
+        securityContext:
+        {{- toYaml . | nindent 10 }}
+        {{- end }}
```

Hi @arueth, would there be any problem if using the following?
```suggestion
        securityContext: {{- toYaml .Values.head.podSecurityContext | nindent 10 }}
```

---

Hi @kevin85421, This PR works and looks good to me. What do you think?